### PR TITLE
Added more Game.cs documentation.

### PR DESCRIPTION
**Pull request by rubiksmaster02**

### CODE REVIEW

**Comment by peppy** [comment]
This looks to be done using automated tooling. The changes are wrong and the additions are mostly misleading or outright incorrect.

**Comment by rubiksmaster02** [comment]
what?

**Comment by rubiksmaster02** [comment]
I literally did this by hand, hold up i can get proof

File: osu.Framework/Game.cs [hunk 1]
```diff
@@ -25,27 +25,45 @@ namespace osu.Framework
 {
     public abstract class Game : Container, IKeyBindingHandler<FrameworkAction>, IHandleGlobalKeyboardInput
     {
+        /// <summary>
+        /// Gets this <see cref="Host"/>'s window.
+        /// </summary>
         public IWindow Window => Host?.Window;
 
+        /// <summary>
+        /// Gets the current ResourceStore{T}.
```
Comment (peppy) [inline osu.Framework/Game.cs:34]: What makes this "current"?

File: osu.Framework/Game.cs [hunk 2]
```diff
@@ -25,27 +25,45 @@ namespace osu.Framework
...
+        /// </summary>
         public ResourceStore<byte[]> Resources { get; private set; }
 
+        /// <summary>
+        /// Gets this <see cref="TextureStore"/>.
```
Comment (peppy) [inline osu.Framework/Game.cs:39]: This is pointless. If we're adding xmldoc it
- doesn't need to say "gets this"
- should probably explain what the property is used for.

File: osu.Framework/Game.cs [hunk 3]
```diff
@@ -25,27 +25,45 @@ namespace osu.Framework
...
+        /// <summary>
+        /// Gets this <see cref="GameHost"/>.
+        /// </summary>
         protected GameHost Host { get; private set; }
 
         private readonly Bindable<bool> isActive = new Bindable<bool>(true);
 
         /// <summary>
-        /// Whether the game is active (in the foreground).
+        /// Gets whether the game is active (in the foreground).
```
Comment (peppy) [inline osu.Framework/Game.cs:51]: Unnecessary

File: osu.Framework/Game.cs [hunk 4]
```diff
@@ -25,27 +25,45 @@ namespace osu.Framework
...
         /// </summary>
         public IBindable<bool> IsActive => isActive;
 
+        /// <summary>
+        /// Get this <see cref="AudioManager"/>.
```
Comment (peppy) [inline osu.Framework/Game.cs:56]: what is "this"? it doesn't make sense.

File: osu.Framework/Game.cs [hunk 5]
```diff
@@ -25,27 +25,45 @@ namespace osu.Framework
...
         public AudioManager Audio { get; private set; }
 
+        /// <summary>
+        /// Get this <see cref="ShaderManager"/>.
+        /// </summary>
         public ShaderManager Shaders { get; private set; }
 
         /// <summary>
-        /// A store containing fonts accessible game-wide.
+        /// Gets a store containing fonts accessible game-wide.
```
Comment (peppy) [inline osu.Framework/Game.cs:66]: unnecessary

File: osu.Framework/Game.cs [hunk 6]
```diff
@@ -64,8 +85,15 @@ public abstract class Game : Container, IKeyBindingHandler<FrameworkAction>, IHa
 
         private LogOverlay logOverlay;
 
+        /// <summary>
+        /// Gets the <see cref="content"/>.
```
Comment (peppy) [inline osu.Framework/Game.cs:89]: content is private. this is incorrect as it should not reference a private variable. also it's pointless.

File: osu.Framework/Game.cs [hunk 7]
```diff
@@ -64,8 +85,15 @@ public abstract class Game : Container, IKeyBindingHandler<FrameworkAction>, IHa
...
+        /// </summary>
         protected override Container<Drawable> Content => content;
 
+        /// <summary>
+        /// Creates a new <see cref="UserInputManager"/>.
```
Comment (peppy) [inline osu.Framework/Game.cs:94]: this should never be called by the user. it's only for overriding purposes. taht's the kind of thing xmldoc should contain.

File: osu.Framework/Game.cs [hunk 8]
```diff
@@ -113,6 +141,11 @@ public virtual void SetHost(GameHost host)
 
         private DependencyContainer dependencies;
 
+        /// <summary>
+        /// Creates a <see cref="DependencyContainer"/> with children.
```
Comment (peppy) [inline osu.Framework/Game.cs:145]: it doesn't "create" more than once. misleading and incorrect (what children?)

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
If this was not a troll pull request hopefully the above is enough to explain.

**Pull request closed without merging**